update_ip/services/nfsn.py

```diff
@@ -1,64 +1,60 @@
+import logging
+
+import tld
+import pynfsn
+
 from update_ip.services.base import BaseDNSService, DNSServiceError
-try:
-    from pynfsn import pynfsn
-except ImportError:
-    raise Exception("This service requires the pynfsn package. You can find it on pypi or github")
 
-import logging
-log= logging.getLogger('update_ip.services.nfsn')
+log = logging.getLogger('update_ip.services.nfsn')
+
+ALLOW_INEXISTENT = True  # If this is false, I'll refuse to proceed if the domain record doesn't already exist
 
-ALLOW_INEXISTENT= True #If this is false, I'll refuse to proceed if the domain record doesn't already exist
 
-def split_domain(domain):
```

Comment:
Yes you're right! for next commits i will do first code-format and later do changes.